src/account/account.entity.ts

```diff
@@ -70,6 +78,17 @@ export class Account extends BaseEntity {
         );
     }
 
+    getApFollowers() {
```

Comment:
🤔 should we be more explicit with the name? `getApFollowersUrl`?

Reply:
IMO The `URL` type communicates that it is a url